packages/visualizations/addon/components/navi-vis-c3-chart.js

```diff
@@ -44,9 +44,12 @@ export default C3Chart.extend({
   dataDidChange() {
     // Add custom classes to each data series for easier reference and coloring
     let { data, dataClasses } = getProperties(this, 'data', 'dataClasses'),
-        dataWithClasses = Ember.assign({}, { classes: dataClasses }, data);
+        dataWithClasses = Ember.assign({}, { classes: dataClasses }, data),
+        dataWithClasses2 = Ember.assign({}, dataWithClasses);
 
-    get(this, 'chart').load(dataWithClasses);
+    //creating a new class for the predicted data line styling
+    dataWithClasses2["classes"]["predicted"] = "chart-series-1 c3-predicted";
```

Comment:
can we show the metric along with the 'predicted' keyword for the class. for example 'Predicted Invalid Revenue' instead of 'predicted'